homeassistant/components/plant/__init__.py

```diff
@@ -282,7 +282,8 @@ async def async_added_to_hass(self):
         """After being added to hass, load from history."""
         if ENABLE_LOAD_HISTORY and "recorder" in self.hass.config.components:
             # only use the database if it's configured
-            self.hass.async_add_job(self._load_history_from_db)
+            self.hass.async_add_executor_job(self._load_history_from_db)
```

Comment:
I think that we should await this, or else the history is not loaded.
```suggestion
            await self.hass.async_add_executor_job(self._load_history_from_db)
```